Example/Pikko/PikkoView.swift

```diff
@@ -22,12 +22,13 @@ class PikkoView: UIView {
     fileprivate func setUpColorPickerViews(_ frame: CGRect) {
         // TODO: Make borderwidth an adjustable parameter.
         let borderWidth: CGFloat = 30.0
+        let selectorDiameter: CGFloat = borderWidth * 1.5
         let radius = frame.width/2
-        let customWidth: CGFloat = sqrt(2) * (radius - borderWidth)
+        let customWidth: CGFloat = sqrt(2) * (radius - borderWidth) * 0.85 // Magic constant that controls "margin" of the square
```

Comment:
introduce constant

Reply:
or write a FIXME to remember that we have to make all view adjustment calculations for the future